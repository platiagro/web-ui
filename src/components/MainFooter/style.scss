.main-footer {
  text-align: center;
}
.ant-layout-footer {
  padding: 1vw 2vw;
}
