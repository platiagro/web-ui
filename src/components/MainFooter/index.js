/* 
  Main footer of the application.
  
  This component is responsible for displaying a copyright message in the main
  application footer.
*/
import React from 'react';

import './style.scss';

const MainFooter = () => {
  const year = new Date().getFullYear();
  const message = `PlatIAgroÂ© ${year} - Todos os direitos reservados`;
  return <div className='main-footer'>{message}</div>;
};

export default MainFooter;
