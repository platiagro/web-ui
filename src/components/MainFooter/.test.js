import React from 'react';

import { shallow } from 'enzyme';

import MainFooter from '.';

describe('MainFooter component', () => {
  it('renders without crashing', () => {
    shallow(<MainFooter />);
  });

  it('renders html correctly', () => {
    const mainFooterShalowed = shallow(<MainFooter />);

    expect(mainFooterShalowed).toMatchSnapshot();
  });
});
