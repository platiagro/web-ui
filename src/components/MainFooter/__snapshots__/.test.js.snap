// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooter component is expected render html correctly 1`] = `
<div
  className="main-footer"
>
  PlatIAgroÂ© 2019 - Todos os direitos reservados
</div>
`;
